app/main.py

```diff
@@ -1 +1,51 @@
-#  write your code here
+from typing import List
+
+
+class Animal:
+
+    def __init__(
+            self,
+            name: str,
+            appetite: int,
+            is_hungry: bool = True
+    ) -> None:
+        self.name = name
+        self.appetite = appetite
+        self.is_hungry = is_hungry
+
+    def print_name(self) -> None:
+        print(f"Hello, I'm {self.name}")
+
+    def feed(self) -> int:
+        if self.is_hungry:
+            print(f"Eating {self.appetite} food points...")
+            self.is_hungry = False
+            return self.appetite
+        else:
```

Comment:
else is useless here

Reply:
return = 0, if I delete this, feed_animals() write:  TypeError: unsupported operand type(s) for +=: 'int' and 'NoneType'

Reply:
I understand, thank you